packages/examples/README.md

```diff
@@ -14,16 +14,7 @@ see https://github.com/eclipse-thingweb/node-wot/issues/171.
 ### Workflow
 
 1. Run `npm run build`
-2. Remove the following 3/4 lines in JS files of folder `dist/`
-
-```
-Object.defineProperty(exports, "__esModule", { value: true });
-require("wot-typescript-definitions");
-let WoT;
-let WoTHelpers;
-```
-
-3. Copy the according JS file(s) to
```

Comment:
The `Object.defineProperty(exports, "__esModule", { value: true });` should be *always* generated (unless you are not importing anything). I'd keep the readme as it was and complete the PR by also copying the files to the dedicated js folder (for demo purposes using plain JS may be even more inclusive). 

Regarding lines 21-23, I'm 90% sure that they are generated only when the script is meant to be used as node-wot script, meaning that WoT and WoTHelpers variables are automatically injected by the CLI context. They are not present in the these quickstart examples because @egekorkan is using node-wot as a library dependency.

Reply:
So those lines are not generated when I run npm run build since WoT variables are injected by the CLI context. I have reverted these lines since we should restructure this folder and put instructions to each readme. Currently, this is in the root folder of the examples and these do not apply to all examples.

I am indeed running node-wot as a dependency in order to create 3 different servient that have different protocols. This way, one has a higher tendency to use node-wot to communicate with 3 Things using different protocols.